src/User.php

```diff
@@ -562,7 +562,13 @@ public static function getUsersIdByEmails(string $email, array $condition = []):
                     ]
                 ]
             ],
-            'WHERE' => [UserEmail::getTable() . '.email' => $email] + $condition
+            'WHERE' =>
+                [
+                    'RAW' => [
+                        'LOWER(' . UserEmail::getTable() . '.email' . ')'  => strtolower($email)
```

Comment:
Email addresses can contain unicode chars. We have to use a method that supports multi-byte strings.
```suggestion
                        'LOWER(' . UserEmail::getTable() . '.email' . ')'  => Toolbox::strtolower($email)
```